File: Level 1/Week03/Day15/hhs/2.java
import java.util.*;

class Solution {
    public int solution(int k, int m, int[] score) {
        Arrays.sort(score);
        int answer = 0;

        for (int i = score.length % m; i < score.length; i += m) {
            int min = k;

            for (int j = i; j < i + m; j++) {
                min = Math.min(min, score[j]);
            }

            answer += min * m;
        }

        return answer;
    }
}

File: Level 2/Week04/Day27/hhs/2.kt
import java.util.Deque
import java.util.LinkedList

class Solution {
    fun solution(s: String): String {
        val answer = StringBuilder()

        val dq: Deque<Char> = LinkedList()

        for (c in s) {
            if (dq.isEmpty()) dq.addLast(c.uppercaseChar())
            else {
                if (c == ' ') {
                    dq.addLast(' ')
                }

                if (c.isLetter() || c.isDigit()) {
                    if (dq.peekLast() == ' ') {
                        dq.addLast(c.uppercaseChar())
                    } else {
                        dq.addLast(c.lowercaseChar())
                    }
                }
            }
        }

        dq.forEach { answer.append(it) }

        return answer.toString()
    }
}